asreview/datasets.py

```diff
@@ -41,9 +55,154 @@ def to_dict(self):
     def __str__(self):
         return pretty_format(self.to_dict())
 
+    def find(self, data_name):
+        if data_name in self.aliases:
+            return self
+        return None
+
+    def list(self, latest_only=True):
+        return [self]
+
+
+class BaseVersionedDataSet():
+    def __init__(self, base_dataset_id, datasets=None, url=None):
```

Comment:
I'm wondering why the arguments of BaseVersionedDataSet and BaseDataSet differ so much. They are similar isn't it

Reply:
Yes, but one is a container for the other!